.github/workflows/deploy.yml

```diff
@@ -428,17 +434,18 @@ jobs:
         name: v${{ steps.tagName.outputs.version }}
         files: |
           Cargo.lock
-          rav1e.exe
-          rav1e-${{ steps.tagName.outputs.version }}-linux-generic.zip
-          rav1e-${{ steps.tagName.outputs.version }}-linux-sse4.zip
-          rav1e-${{ steps.tagName.outputs.version }}-linux-avx2.zip
-          rav1e-${{ steps.tagName.outputs.version }}-linux-aarch64.tar.gz
-          rav1e-${{ steps.tagName.outputs.version }}-macos.zip
-          rav1e-${{ steps.tagName.outputs.version }}-windows-msvc-generic.zip
-          rav1e-${{ steps.tagName.outputs.version }}-windows-msvc-sse4.zip
-          rav1e-${{ steps.tagName.outputs.version }}-windows-msvc-avx2.zip
-          rav1e-${{ steps.tagName.outputs.version }}-windows-gnu-generic.zip
-          rav1e-${{ steps.tagName.outputs.version }}-windows-gnu-sse4.zip
-          rav1e-${{ steps.tagName.outputs.version }}-windows-gnu-avx2.zip
+          rav1e (Windows-msvc-generic)/rav1e.exe
+          rav1e-ch (unstable) (AVX2)/rav1e-ch.exe
```

Comment:
Do we want to start including `rav1e-ch.exe` on releases although channel API is still under `unstable`?

Reply:
we can and hopefully I'll find time to see if we can overhaul it a bit to avoid the corner cases with a thread pool too small.